botocore/compat.py

```diff
@@ -237,3 +242,33 @@ def total_seconds(delta):
     day_in_seconds = delta.days * 24 * 3600.0
     micro_in_seconds = delta.microseconds / 10.0**6
     return day_in_seconds + delta.seconds + micro_in_seconds
+
+
+# Checks to see if md5 is available on this system. A given system might not
+# have access to it for various reasons, such as FIPS mode being enabled.
+try:
+    hashlib.md5()
+    MD5_AVAILABLE = True
+except ValueError:
+    MD5_AVAILABLE = False
+
+
+def get_md5(raise_error_if_unavailable=True, *args, **kwargs):
```

Comment:
After discussing, we can remove the `raise_error_if_unavailable` argument implemented for the CLI and update the CLI code to not compute MD5's if it's not possible.